artemis-server/src/main/java/org/apache/activemq/artemis/core/transaction/impl/CoreTransactionDetail.java

```diff
@@ -56,6 +56,6 @@ public String decodeMessageType(Message msg) {
 
    @Override
    public Map<String, Object> decodeMessageProperties(Message msg) {
-      return msg.toMap();
+      return msg.toMap(256);
```

Comment:
Surely this could break existing case. I would expect this to remain unlimited

Reply:
Or at least configurable with default of unlimited

Reply:
on the default. I want the broker to be constrained by default but yes the focus is the management api b/c that returns all the data. I want to fix it such that the broker returns 'some' reasonable amount of data  to ui/management/control by default.

Reply:
Yes but this is decode and isnt just for ui....

Reply:
yep. that is back to the original